hat/assets/js/apps/Iaso/domains/orgUnits/groups/hooks/requests.js

```diff
@@ -17,6 +17,16 @@ export const useGetGroups = params => {
     if (params.search) {
         newParams.search = params.search;
     }
+    if (params.project_ids) {
+        newParams.projectIds = params.project_ids;
+    }
+    if (params.dataSource) {
+        newParams.dataSource = params.dataSource;
+        if (params.version) {
```

Comment:
On my humble opinion there is not need of nesting the if(params.version) inside params.dataSource check